packer/windows.pkr.hcl

```diff
@@ -0,0 +1,154 @@
+variable "image_date" { type = string }
+variable "gcp_project" { type = string }
+variable "task_name" { type = string }
+
+variable "prefix" {
+  type = string
+  default = ""
+}
+
+locals {
+  name = "${var.prefix}pg-ci"
+
+  windows_gcp_images = [
+    {
+      name = "${var.task_name}"
+    },
+  ]
+}
+
+source "googlecompute" "windows-vm" {
+  disk_size               = "50"
+  disk_type               = "pd-ssd"
+  project_id              = var.gcp_project
+  source_image_family     = "windows-2022"
+  image_name              = "${local.name}-${var.task_name}-${var.image_date}"
+  image_family            = "${local.name}-${var.task_name}"
+  zone                    = "us-west1-a"
+  machine_type            = "n2-standard-4"
+  instance_name           = "build-${var.task_name}-${var.image_date}"
+  communicator            = "winrm"
+  winrm_username          = "packer_user"
+  winrm_insecure          = true
+  winrm_use_ssl           = true
+  winrm_timeout           = "10m"
+  metadata = {
+    windows-startup-script-cmd = "winrm quickconfig -quiet & net user /add packer_user & net localgroup administrators packer_user /add & winrm set winrm/config/service/auth @{Basic=\"true\"}"
```

Comment:
Do we need to remove that user?

Reply:
I am not sure if that is possible, because we are connecting to the image as this user. Also, [Cirrus images have this user](https://github.com/cirruslabs/vm-images/blob/master/googlecompute/windows_images.json#L21), too.